Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -14972,7 +14964,6 @@
             <LayoutButtons Face="Attack" Type="AbilCmd" AbilCmd="attack,Execute" Row="0" Column="4"/>
             <LayoutButtons Face="AcquireMove" Type="AbilCmd" AbilCmd="move,AcquireMove" Row="0" Column="4"/>
             <LayoutButtons Face="Rally" Type="AbilCmd" AbilCmd="ProgressRally,Rally1" Row="2" Column="4"/>
-            <LayoutButtons Face="AP_AWrp" Type="AbilCmd" AbilCmd="AP_ArchonWarp,SelectedUnits" Row="2" Column="3"/>

```

Comment:
I'm restoring this as it's a part of basic AA logic. Could be gated behind an item in the future